### PR TITLE
Vizlab #348 create content.css

### PR DESCRIPTION
**Pull request by mwernimont**

### CODE REVIEW

File: inst/resource.library.yaml [hunk 1]
```diff
@@ -104,9 +104,10 @@
     vizlab-js: lib-vizlab-javascript
     svg-injector-js: lib-svg-injector-js
     favicon: lib-vizlab-favicon
+    content-CSS: lib-content-css
```
Comment (aappling-usgs) [inline inst/resource.library.yaml]: could we keep it all lower case?

**Comment by aappling-usgs** [comment]
uh oh, this looks like a new testthat error:
```
> test_check("vizlab")
── 1. Error: vizmake creates makefile and runs make (@tests-remake.R#46)  ──────
While processing target 'header-template':
    Expected all elements to be scalar
1: expect_message(regexp = "Starting build at", expect_warning(vizmake("plot_info"), 
       "these packages are newer than required")) at testthat/tests-remake.R:46
2: quasi_capture(enquo(object), capture_messages, label = label)
3: capture(act$val <- eval_bare(get_expr(quo), get_env(quo)))
4: withCallingHandlers(code, message = function(condition) {
       out$push(condition)
       invokeRestart("muffleMessage")
   })
5: eval_bare(get_expr(quo), get_env(quo))
6: expect_warning(vizmake("plot_info"), "these packages are newer than required")
7: quasi_capture(enquo(object), capture_warnings, label = label)
8: capture(act$val <- eval_bare(get_expr(quo), get_env(quo)))
9: withCallingHandlers(code, warning = function(condition) {
       out$push(condition)
       invokeRestart("muffleWarning")
   })
10: eval_bare(get_expr(quo), get_env(quo))
...
18: FUN(X[[i]], ...)
19: FUN(nX[[i]], X[[i]], ...)
20: withCallingHandlers(make_target_dat(dat), error = catch_error_prefix(prefix), warning = catch_warning_prefix(prefix))
21: make_target_dat(dat)
22: process_target_command(name, dat, file_extensions)
23: unlist(from_yaml_map_list(dat$depends))
24: from_yaml_map_list(dat$depends)
25: stop("Expected all elements to be scalar")
26: .handleSimpleError(function (e) 
   {
       e$message <- paste0(prefix, e$message)
       stop(e)
   }, "Expected all elements to be scalar", quote(from_yaml_map_list(dat$depends)))
27: h(simpleError(msg, call))
```

**Comment by mwernimont** [comment]
So just did the vizlab update to the water-use and the build broke, so thats probably the problem?  Will fix and PR it.

**Comment by mwernimont** [comment]
It's helpful if the repo has access to the images you want it to use...

**Comment by aappling-usgs** [comment]
:crossed_fingers:

**Comment by ldecicco-USGS** [comment]
Can you add a div in printAll with some style?

perhaps:
```
<div id="{{id}}">
  <div class="textContentAligner">
    <p>
      {{{ text }}}
    </p>
  </div>
</div>
```
and in content:
```
.textContentAligner{
  padding:0 10px;
}
```

**Comment by mwernimont** [comment]
Was gonna do that in a different PR for Text has no margin #342

**Comment by ldecicco-USGS** [comment]
kewl

**Comment by aappling-usgs** [comment]
Success!

**Pull request merged**